<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>device_install_dialog</class>
 <widget class="QDialog" name="device_install_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>264</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Device install wizard</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="2" column="0">
    <widget class="QWidget" name="rpkg_browse_widget" native="true">
     <layout class="QHBoxLayout" name="rpkg_browse_layout">
      <item>
       <widget class="QLabel" name="rpkg_edit_name_label">
        <property name="text">
         <string>RPKG</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rpkg_path_line_edit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rpkg_browse_btn">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="installation_choose_widget" native="true">
     <layout class="QHBoxLayout" name="installation_choose_layout" stretch="0,0">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="installation_choose_label">
        <property name="text">
         <string>Choose the installation method</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="installation_combo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Device dump (RPKG + ROM)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Firmware (VPL)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QWidget" name="opinion_widget" native="true">
     <layout class="QHBoxLayout" name="confirmation_layout">
      <item>
       <widget class="QPushButton" name="confirmation_install_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Install</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="confirmation_cancel_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="rom_browse_widget" native="true">
     <layout class="QHBoxLayout" name="rom_browse_layout">
      <item>
       <widget class="QLabel" name="rom_edit_name_label">
        <property name="text">
         <string>ROM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rom_path_line_edit">
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rom_browse_btn">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="vpl_browse_widget" native="true">
     <layout class="QHBoxLayout" name="vpl_browse_layout">
      <item>
       <widget class="QLabel" name="vpl_edit_name_label">
        <property name="text">
         <string>VPL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="vpl_path_line_edit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="vpl_browse_btn">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QProgressBar" name="install_progress_bar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
